test/domino.js

```diff
@@ -706,6 +706,15 @@ exports.gh70 = function() {
   classAttr.value.should.equal('textContent');
   classAttr.nodeValue.should.equal('textContent');
   classAttr.textContent.should.equal('textContent');
+  classAttr.innerText.should.equal('textContent');
+};
+
+exports.innerText = function() {
+  var document = domino.createDocument('<h1></h1>');
```

Comment:
I'd propose adding an extra test to show how innerText works with nested elements (both setter and getter), for example:
```
exports.innerText = function() {
  var document = domino.createDocument('<div>Hello <h1>world!</h1></div>');
  var div = document.querySelector('div');
  div.innerText.should.equal('Hello world!'); // getter
  div.innerText = 'Hi'; // setter
  div.textContent.should.equal('Hi');
  div.innerText.should.equal('Hi');
};
```